### ISSUE TITLE
The "sticky" option doesn't seem to work

### ISSUE THREAD

**Issue by Penguin-jpg** [numToStr/Comment.nvim#487]
I tested the sticky option and found that it didn't affect the cursor position, whether it was true or false.
Below is my config for Comment.nvim:
![image](https://github.com/user-attachments/assets/786a00cb-e6a1-4ae5-bd4c-584cfda607f7)
(I am using neovim 0.10.0 and the latest version of Comment.nvim .)

What I want to achieve is something like this:
```
-- "^" means the current cursor position.

-- Before comment
apple
^
-- After comment
-- apple
   ^
```

What I encounter:
```
-- Before comment
apple
^
-- After comment
-- apple
^
```

**Comment by windowsrefund** [comment]
Seems to be 2 issues here which look quite different depending on whether a line comment was used or something like a `gcap` or`gcip`. As mentioned above, the cursor will maintain the original line+column position on when a `gcc` is used.

The same is true when a `gcip` is used. However, an immediate undo `u` will jump the cursor to the first line and first column of the paragraph (this was not the original position of the cursor).

Either way, we're clearly not sticking but I figure this "jump" behavior is worth calling out.

**Comment by Penguin-jpg** [comment]
Thank you for the explanation. I think maybe I misunderstands the meaning the sticky option. I'm sorry for the ambiguity. I think what I use in the example is `gcc`.
What I want to achieve is more like the comment function in vscode, where the cursor stays at the character I am hovering, not the original position of the cursor.

**Comment by windowsrefund** [comment]
Yep, I'd like that as well but could live with it persisting the original position if needed. What I do find utterly annoying is the "jump" I've described when performing against an object other than a simple line. Hopefully this will be an easy fix.
